src/sentry/api/serializers/models/organization.py

```diff
@@ -109,11 +110,22 @@ def serialize(self, obj, attrs, user):
 
 
 class OnboardingTasksSerializer(Serializer):
+    def get_attrs(self, item_list, user, **kwargs):
+        # Unique user list
+        users = {item.user_id: item.user for item in item_list if item.user}.values()
```

Comment:
`users = {item.user for item in item_list if item.user}`?

Reply:
Does this uniqeify correctly?

Reply:
Yeah, it'll use a set rather than a dict.

Reply:
Fixed